test/smoke/src/areas/positron/plots/plots.test.ts

```diff
@@ -398,6 +395,13 @@ tree`;
 
 				await simplePlotTest(app, script, '.jstree-container-ul');
 
+				// tree should be expanded by default
+				const treeNodes = app.workbench.positronPlots.getWebviewPlotLocator('.jstree-container-ul .jstree-node');
+				await expect(treeNodes).toHaveCount(9);
+
+				// collapse the tree, only parent nodes should be visible
+				treeNodes.first().click({ position: { x: 0, y: 0 } }); // target the + icon
+				await expect(treeNodes).toHaveCount(3);
```

Comment:
Not sure why.. but on my box (Windows 11), something about this causes the next test to fail.  I verified this by running the tests on `main` and they pass without issue.

Reply:
Oh weird, so it's just _your_ machine, but not CICD land. Usually it's the opposite. 😂

Reply:
Ha! It's probably a windows issue... something about how python on windows interprets whitespace/tabs, etc.